client/packages/common/src/hooks/useDialog/useDialog.tsx

```diff
@@ -224,7 +234,20 @@ export const useDialog = (dialogProps?: DialogProps): DialogState => {
         sx={sx}
         TransitionComponent={Transition}
         disableEscapeKeyDown={false}
+        fullScreen={fullScreen}
       >
+        {fullScreen && (
+          <IconButton
```

Comment:
Some modals used clicking on the backdrop to close (e.g. internal order create) so added close button